core/templates/dev/head/components/OutcomeEditorDirective.js

```diff
@@ -63,6 +67,21 @@ oppia.directive('outcomeEditor', [function() {
           onExternalSave();
         });
 
+        $scope.getCurrentInteractionId = function() {
```

Comment:
@seanlip Yeppers. It shows up as undefined.

Looks like this in AnswerGroupEditorDirective Scope
suppressWarnings: '&',

Passed in like this to answer-group-editor tag in state_editor_responses.html
suppress-warnings="suppressDefaultAnswerGroupWarnings()"

Looks like this in OutcomeEditorDirective scope
suppressWarnings: '&suppressWarnings'

Passed in like this to outcome-editor tag in answer_group_directive.html
suppress-warnings="suppressDefaultAnswerGroupWarnings()"